packages/base-sheets/src/commands/commands/__tests__/set-tab-color.command.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { ICommandService, ICurrentUniverService, Univer } from '@univerjs/core';
+import { Injector } from '@wendellhu/redi';
+import { afterEach, beforeEach, describe, expect, it } from 'vitest';
+
+import { SetTabColorMutation } from '../../..';
+import { SetTabColorCommand } from '../set-tab-color.command';
+import { createCommandTestBed } from './create-command-test-bed';
+
+describe('Test tab color commands', () => {
+    let univer: Univer;
+    let get: Injector['get'];
+    let commandService: ICommandService;
+
+    beforeEach(() => {
+        const testBed = createCommandTestBed();
+        univer = testBed.univer;
+        get = testBed.get;
+
+        commandService = get(ICommandService);
+        commandService.registerCommand(SetTabColorCommand);
+        commandService.registerCommand(SetTabColorMutation);
+    });
+
+    afterEach(() => {
+        univer.dispose();
+    });
+
+    describe('Set several specific colors', () => {
+        function getTabColor() {
+            return get(ICurrentUniverService)
+                .getUniverSheetInstance('test')
+                ?.getWorkBook()
+                ?.getActiveSheet()
+                ?.getTabColor();
+        }
+        describe('correct situations', () => {
+            it('will set tab color to #cccccc', async () => {
```

Comment:
Add blank lines around test cases to make the code more readable.

Reply:
done